arroyo/processing/strategies/batching.py

```diff
@@ -116,6 +116,7 @@ def submit(self, message: Message[ValuesBatch[TPayload]]) -> None:
         if self.__batch_to_send:
             raise MessageRejected
 
+        assert not isinstance(message.payload, FilteredPayload)
```

Comment:
Why asserting this is not a filtered payload? Does the unbatch job cares if it receives a filtered message? 
It seems to me it should just ignore them.

Reply:
I assume `BatchStep` and `UnbatchStep` always come in pairs. `BatchStep` should never produce a `FilteredMessage`.

I just realized however one could theoretically put a filter step between both of them, so the outer message could be filtered.

I updated the strategy to deal with filtered messages "normally"